### PR TITLE
fix(DTFS2-6759): fix token expiry

### PR DESCRIPTION
**Pull request by AlexBramhill**
# Introduction
Currently, token expiry is non-functional. We are setting the token expiry based on Date.now() which uses milliseconds since epoch. JWTs work with seconds since epoch ([RFC 7519: JSON Web Token (JWT) (rfc-editor.org)](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.6))

# Resolution
This fix removes setting `ias` as it is unneeded. 

# Misc
I have checked to see what happens when a token is no longer valid in `portal-api`. Portal logs the user out and their session data is deleted.

### CODE REVIEW

**Comment by AlexBramhill** [comment]
Note: I wont merge this until I've merged [this ticket](https://github.com/UK-Export-Finance/dtfs2/pull/2198).

File: portal-api/src/crypto/utils.js
Comment (lmarrai-sw) [inline portal-api/src/crypto/utils.js:64]: Just to double-check I've understood:
- `jsonwebtoken` was calculating the `exp` claim from the `iat` and `expiresIn` that we specified (and the `iat` was out by a factor of 1000 so our `exp` claim was incorrect)
- by not providing the `iat` ourselves:
  - `jsonwebtoken` will calculate `exp` from `expiresIn` correctly
  - `jsonwebtoken` will still calculate the `iat` claim for the JWT itself and include it

Is that right?

**Review by lmarrai-sw — APPROVED** [review]

File: portal-api/src/crypto/utils.js
Comment (AlexBramhill) [reply portal-api/src/crypto/utils.js:64]: Exactly 
https://github.com/auth0/node-jsonwebtoken

Generated jwts will include an `iat` (issued at) claim by default unless `noTimestamp` is specified. If `iat` is inserted in the payload, it will be used instead of the real timestamp for calculating other things like `exp` given a timespan in `options.expiresIn`.

**Pull request merged**
